lavalink/PlayerManager.py

```diff
@@ -229,18 +229,12 @@ def get(self, guild_id):
 
         return self._players[guild_id]
 
-    async def clear_guild(self, guild_id):
-        """ Removes the player and clears the cache. """
-        try:
-            player = self._players.pop(guild_id)
-            await player.disconnect()
-        except KeyError:
-            pass
-
-    async def remove(self, guild_id):
+    async def remove(self, guild_id, call_cleanup: bool=True):
         """ Removes a player from the current players. """
         if guild_id in self._players:
             player = self._players.pop(guild_id)
+            if call_cleanup is True:

```

Comment:
Do you really need to specify if it's True ?

Reply:
Probably worth specifying type for `guild_id` too for consistency with `call_cleanup`